fn_ipvoid/setup.py

```diff
@@ -0,0 +1,33 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+setup(
+    name='fn_ipvoid',
+     version='1.0.0',
+    license='MIT',
+    author='IBM Resilient Labs',
+    author_email='resil.labs@gmail.com',
+    url='https://github.com/ibmresilient/resilient-community-apps',
+    description="Resilient Circuits Components for 'fn_ipvoid'",
+    long_description="Resilient Circuits Components for 'fn_ipvoid'",
+    install_requires=[
+        'resilient_circuits>=30.0.0'
+    ],
+    packages=find_packages(),
+    include_package_data=True,
+    platforms='any',
+    classifiers=[
+        'Programming Language :: Python',
+    ],
+    entry_points={
+        "resilient.circuits.components": [
+            "FnIpvoidFunctionComponent = fn_ipvoid.components.function_ipvoid:FunctionComponent"
```

Comment:
Function should be called `fn_ip_void_request`.

Function name should not be the same name as the package name. Function and package name are defined in Trello Ticket: https://trello.com/c/vpRNTxqO/24-sudip-ip-void

The message destination name can be the same as the package name